legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -139,8 +149,12 @@ def validate_roles(incorporation_json) -> Error:
 
 def validate_parties_mailing_address(incorporation_json) -> Error:
     """Validate the person data of the incorporation filing."""
+    legal_type = incorporation_json['filing']['incorporationApplication']['nameRequest']['legalType']
```

Comment:
If you are trying to get legalType of business get it from '/filing/business/legalType' and I would suggest to use get_str function

Reply:
updated to grab legal type from `/filing/business/legalType` and use `get_str` function